package.json

```diff
@@ -13,7 +13,7 @@
     "start": "NODE_ENV=production  node dist/index.js",
     "build": "babel src -d dist",
     "prestart": "NODE_ENV=production sequelize db:migrate",
-    "pretest": "cross-env NODE_ENV=test npm run migrate:reset && cross-env NODE_ENV=test  npm run migrate && NODE_ENV=test npm run seed"
```

Comment:
Please is there any reason why you removed `NODE_ENV=test` from the second and third ampersand(&) of this pretest script?

Reply:
It seemed redundant given we've set the environment to test before calling both npm scripts. Except I'm missing something. try running it in this format at your end and let me know if it breaks your system.

Reply:
For Windows users, it defaults back to `development` after each (&&)